python/cuspatial/cuspatial/tests/test_geodataframe.py

```diff
@@ -321,7 +321,7 @@ def test_boolmask(gpdf, df_boolmask):
 
 
 def test_memory_usage(gs):
-    assert gs.memory_usage() == 224
+    assert gs.memory_usage() == 228
```

Comment:
Since this would be `xfail`ed we can revert back to the original code

```suggestion
    assert gs.memory_usage() == 224
```